content/jobs/2022/product-community-lead.md

```diff
@@ -82,4 +83,8 @@ Below are several skills that will make somebody well-suited for this role. Ther
 * Experience **using an open source language** in a research/education setting. 
 * An interest in learning about **workflows with the open source language Python**, and open source workflows around it for research and education.
 * Experience or strong interest in **cloud infrastructure and data workflows** for research or education, and how this can lead to more collaboration, openness, and productivity (for example, via managed services and [Analysis Ready, Cloud Optimized datasets](https://eartharxiv.org/repository/view/2726/)).
-* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.
\ No newline at end of file
+* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.
+
+## A note on scope and expectations
+
+We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c. We will do everything we can to set these expectations intentionally, and to refine them as we learn more, in order to reduce this ambiguity and ensure that this role is sustainable.
```

Comment:
What do you think @damianavila - good point about not under-selling ourselves :-)

Reply:
Super! Thanks!